src/main/java/org/jenkinsci/plugins/ewm/nodes/NodeDisk.java

```diff
@@ -24,12 +24,12 @@
 public class NodeDisk implements Describable<NodeDisk> {
 
     private final String diskRefId;
-    private final String localRootPath;
+    private final String nodeMountPoint;
```

Comment:
I think that we'll need a `readReslve()` method here to maintain backwards compatibly. Probably there aren't many users who configured the plugin yet, but still.
It's not that difficult, here are some details https://wiki.jenkins-ci.org/display/JENKINS/Hint+on+retaining+backward+compatibility, at the _Scenario: Rename a field_

So in our case, I think we have the following:
```java
// keep this field for backward compatibility
private transient String localRootPath;
private final String nodeMountPoint;

// ...

protected Object readResolve() {
    if (localRootPath != null) {
        nodeMountPoint = localRootPath;
    }
    return this;
}
```
But I'm not very sure if this is the right way to do it.
Maybe @oleg-nenashev can give us more details.